devcycle_python_sdk/open_feature_provider/provider.py

```diff
@@ -138,9 +138,11 @@ def resolve_float_details(
     def resolve_object_details(
         self,
         flag_key: str,
-        default_value: Union[dict, list],
+        default_value: Union[Mapping[str, FlagValueType], Sequence[FlagValueType]],
```

Comment:
does this break backwards compatibility?

Reply:
yeah, we are extending types instead of being strict about dict and list with Mapping and Sequence